### PR TITLE
Information on sending many transactions

### PR DESCRIPTION
**Pull request by bkase**
Covers details around:
1. Rate limiting
2. Recovering from crashing
3. Cancelling transactions

### CODE REVIEW

File: pages/en/using-mina/send-payment.mdx [hunk 1]
```diff
@@ -142,3 +142,25 @@ You may not see the `Receipt chain hash` on the first transaction from the accou
...
+
+### Sending Many Transactions
+
+Sometimes you may wish to send many transactions: For example, to payout rewards to those delegating to you if you're running a staking pool.
+
+All information here is relevant as of the 1.2.0 build (currently in beta):
+
+### Rate limiting
+
+Currently, nodes on the network will rate limit receiving messages from a given node. As of the 1.2.0 build, your node will also follow this rate limit when sending transactions. Specifically, the limit is currently set at 10 transactions every 15 seconds computed over a 5 minute window. If you attempt to send transactions faster than this rate, your node will queue them up and flush them as older transactions expire from the window upon which the rate limit is computed. You do not need to throttle sending these transactions yourself.
```
Comment (mrmr1993) [inline pages/en/using-mina/send-payment.mdx:156]: `Note that older releases of the mina daemon do not perform this rate limiting; if you are running an older version, you should manually limit the number of transactions. Due to overheads from rebroadcasting transactions, we do not recommend sending more than 50 transactions every 5 minutes if you need to manually rate limit.`

**Review by mrmr1993 — APPROVED** [review]
Suggested an addition about old versions. LGTM.

**Comment by jasongitmail** [comment]
Added some minor grammar clean up. LGTM

File: pages/en/using-mina/send-payment.mdx [hunk 2]
```diff
@@ -101,7 +101,7 @@ Tip: You can run `mina client status` to see the current block height updating.
 
 ## Make a payment
 
-Finally we get to the good stuff, sending our first transaction! Before you send a payment, you'll need to unlock your account:
+Finally, we get to the good stuff–sending our first transaction! Before you send a payment, you'll need to unlock your account:
```
Comment (mrmr1993) [inline pages/en/using-mina/send-payment.mdx:104]: This hyphen reads as a compound noun 'stuff-sending'. Can you replace it with a colon, or add spaces?
Typographically, I think this should also be an em-dash; using that symbol directly, or the common alternative of `--` would be better.

Comment (jasongitmail) [reply pages/en/using-mina/send-payment.mdx:104]: It's already an em dash. But we can use the HTML entity to see if it shows differently on the website. `&mdash;`

Comment (jasongitmail) [reply pages/en/using-mina/send-payment.mdx:104]: Actually, it looks ok on the preview: https://deploy-preview-65--mina-docs.netlify.app/en/using-mina/send-payment  But LMK if you prefer `&mdash;`

Comment (mrmr1993) [reply pages/en/using-mina/send-payment.mdx:104]: So it is, sorry. I was fooled by the monospaced font 🤦‍♂️

Comment (jasongitmail) [reply pages/en/using-mina/send-payment.mdx:104]: Yeah, I was baffled in VS Code for a moment before realizing the same earlier lol

**Pull request merged**